azext_iot/central/params.py

```diff
@@ -179,6 +219,6 @@ def load_deprecated_params(self, _):
         context.argument("timeout", arg_type=event_timeout_type)
         context.argument("properties", arg_type=event_msg_prop_type)
         context.argument(
-            "module_id", options_list=["--module-id", "-m"], help="Iot Edge Module ID",
+            "module_id", options_list=["--module-id", "-m"], help="IoT Edge Module ID",
```

Comment:
nit - lets keep deprecated/non deprecated help texts in line

"module_id", options_list=["--module-id", "-m"], help="Provide IoT Edge Module ID if the device type is IoT Edge.",

vs


            "module_id", options_list=["--module-id", "-m"], help="Iot Edge Module ID",	            "module_id", options_list=["--module-id", "-m"], help="IoT Edge Module ID",

should be the same

Reply:
It's almost the end of the year, and seeing the `# TODO: Delete this by end of Dec 2020` gets me wondering if that is happening.

Reply:
This on my radar will be pulling it in next week.